env/production/aws-iam-policy-NextstrainPathogen@.tf

```diff
@@ -70,6 +70,22 @@ resource "aws_iam_policy" "NextstrainPathogen" {
           "arn:aws:s3:::nextstrain-staging/files/datasets/${each.key}/*",
         ],
       },
+      {
+        "Sid": "CloudFront",
+        "Effect": "Allow",
+        "Action": [
+          "cloudfront:ListDistributions",
```

Comment:
`cloudfront:ListDistributions` [doesn't allow limiting by distribution](https://docs.aws.amazon.com/service-authorization/latest/reference/list_amazoncloudfront.html#amazoncloudfront-ListDistributions); it only supports all or nothing. I misread the table of supported condition keys by actions for it.